.github/pull_request_template.md

```diff
@@ -6,16 +6,11 @@
 --- Merge policy ---
 
 - [ ] Travis CI PASS
-- [ ] `*-aws-runtest` PASS
-- [ ] `*-azure-runtest` PASS
-- [ ] `*-images-runtest` PASS
-- [ ] `*-openstack-runtest` PASS
-- [ ] `*-vmware-runtest` PASS
+- [ ] `cockpit/*/TestSanity` PASS
+- [ ] `cockpit/*/TestImages` PASS
+- [ ] `cockpit/*/TestAWS` PASS
+- [ ] `cockpit/*/TestAzure` PASS
+- [ ] `cockpit/*/TestOpenStack` PASS
+- [ ] `cockpit/*/TestVmware` PASS
```

Comment:
I think we can leave those out now: GitHub show the status and even allows marking some tests as `required`.